reference/tiflash/maintain.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Maintain a TiFlash Cluster
+summary: Learn common operations when you maintain a TiFlash cluster.
+category: reference
+---
+
+# Maintain a TiFlash Cluster
+
+This document describes common operations when you maintain a TiFlash cluster, including checking the version, node logout, troubleshooting, critical logs, and a system table.
+
+## Check the TiFlash version
+
+There are two ways to check the TiFlash version:
+
+- If the binary file name of TiFlash is `tiflash`, you can check the version by executing the `./tiflash version` command.
+
+    However, to execute the above command, you need to add the directory path which includes the `libtiflash_proxy.so` dynamic library to the `LD_LIBRARY_PATH` environment variable. This is because the running of TiFlash relies on the `libtiflash_proxy.so` dynamic library.
+
+    For example, when `tiflash` and `libtiflash_proxy.so` are in the same directory, you can first switch to this directory, and then use the following command to check the TiFlash version:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    LD_LIBRARY_PATH=./ ./tiflash version
+    ```
+
+- Check the TiFlash version by referring to the TiFlash log. For the log path, see the [logger] part in [Configure the `tiflash.toml` file](/reference/tiflash/configuration.md#configure-the-`tiflash.toml`-file). For example:
```

Comment:
```suggestion
- Check the TiFlash version by referring to the TiFlash log. For the log path, see the `[logger]` part in [the `tiflash.toml` configuration file](/reference/tiflash/configuration.md#configure-the-`tiflash.toml`-file). For example:
```
The anchor link is wrong. Please correct it by referring to the PingCAP documentation style guide. @toutdesuite